applications/dotwise.md

```diff
@@ -0,0 +1,191 @@
+
+  **Fast API:** FastAPI is a modern, fast (high-performance), web framework for building APIs with Python 3.7+ based on standard Python type hints. That is fully compatible with OpenAPI 3.0 (formerly known as Swagger). It is a great tool for building APIs with Python.
+
+- Documentation of core components, protocols, architecture, etc. to be deployed
+  - General Guides & Documentation
+  - Test Coverage Reports
+
+- What your project is _not_ or will _not_ provide or implement
+  - There isn't any support for parachains that SubQuery doesn't support
+  - SQL based queries not supported, only GraphQL
+
+### Ecosystem Fit
+
+- Where and how does your project fit into the ecosystem?
+
+  Polkadot ecosystem doesn't have a solution like Dune Analytics and this projects aims start to fill this gap. Users can explore and analyze the data on the Polkadot & Kusama networks with custom queries and visualizations easily and share their queries or dashboards with the community.
+
+- Who is your target audience (parachain/dapp/wallet/UI developers, designers, your own user base, some dapp's userbase, yourself)?
+
+  Who needs to query and visualize Polkadot on-chain data easily is target base of Dotwise.
+
+- What need(s) does your project meet?
+
+  - Users should be able to query and visualize the data on the Polkadot & Kusama networks easily. 
+  - Users can create graphql queries using the built-in query builder.
+  - Users who are not familiar with GraphQL can use the built-in query builder to create queries just use the select boxes and input fields easily.
+
+- Are there any other projects similar to yours in the Substrate / Polkadot / Kusama ecosystem?
+  
+  No, there isn't any similar project in the Polkadot ecosystem but there is developing solutions about data analysis as my knowledge. So if there will be any good data source ı can rich the my data sources and make the platform more useful.
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Tolga Yaycı
+
+### Contact
+
+- **Contact Name:** Tolga Yaycı
+- **Contact Email:** [tolgayayci@protonmail.com](mailto:tolgayayci@protonmail.com)
+- **Website:** [LinkedIn](https://www.linkedin.com/in/tolgayayci/)
+
+### Legal Structure
+
+- **Registered Address:** -
+- **Registered Legal Entity:** -
+
+### Team's experience
+
+As a full-stack developer with 2 years of experience, I have honed my skills in software development, with a focus on dApp development in the past year. I have a deep interest in the Web3 and NFT space and have put my skills to the test by creating a number of relevant applications. In addition to my experience, I have developed detailed React and Next.js projects, further enhancing my ability to build robust and scalable web applications.
```

Comment:
I do not have a project that I have developed on data analysis on Substrate and I am making my first application on this subject. I have been interesting with Polkadot for a while and I aim to develop an effective data analysis application with this project. 

I decided to use SubQuery as the basis for this, because it helps me develop a system that will serve the purposes of the project quickly according to my research and experimentation. As I stated, after v1 completed I will consider to add other data sources as well.